DllImportGenerator/TestAssets/NativeExports/NativeExports.csproj

```diff
@@ -8,28 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="DNNE" Version="1.0.11" />
+    <PackageReference Include="DNNE" Version="1.0.12" />
   </ItemGroup>
 
-  <!--
```

Comment:
I added this workaround to DNNE. There is a property to avoid adding it if we find any issues - https://github.com/AaronRobinsonMSFT/DNNE/blob/4a21add7d8fc7e161f11f523a281122e1f8d2342/src/msbuild/DNNE.props#L45-L46.